CONTRIBUTING.rst

```diff
@@ -107,20 +107,20 @@ The implementation named `<implementation>`, located under `abel/<implementation
  -  ``<implemenation>_transform`` :  core transform (when defined)
  -  ``_bs_<implementation>`` :  function that generates  the basis sets (if necessary)
  -  ``bs_<implementation>_cached`` : function that loads the basis sets from disk, and generates them if they are not found (if necessary).
```

Comment:
`bs_<implementation>_cached` is actually a common function `get_bs_cached(method, ...)` located in `abel.tools.basis.py`, for Dasch and Gerber methods, and for `basex` the function `abel.basex.get_bs_basex_cached()`. The `basex` version may easily be incorporated into `basis.py` (once I, or someone else has a little spare time).  The reason for `get_bs_cached` is just the way it was done in `basex`.

So, perhaps line 109 may read:
```
``abel.tools.basis.py`` (if necessary) modify this file to provide for loading the basis sets from disk, or calling the generator function
```